### PR TITLE
feat: uninstall avd

### PR DESCRIPTION
**Pull request by itsspriyansh**
### Description 

This PR adds the support for deleting an AVD. This feature can be accessed with this command: 
```bash
npx @nightwatch/mobile-helper android uninstall --avd
```

[Screencast from 02-08-24 11:59:10 AM IST.webm](https://github.com/user-attachments/assets/08adc91e-5dd3-4f2e-ab5c-5ed745e33efe)

### CODE REVIEW

**Comment by itsspriyansh** [comment]
Prerequisite #59 for adding the `index.ts` file

File: src/commands/android/subcommands/uninstall/avd.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    const avdmanagerLocation = getBinaryLocation(sdkRoot, platform, 'avdmanager', true);
+    if (!avdmanagerLocation) {
+      showMissingBinaryHelp('avdmanager');
+
+      return false;
+    }
+
+    const installedAvds = execBinarySync(avdmanagerLocation, 'avdmanager', platform, 'list avd -c');
+    if (!installedAvds) {
+      Logger.log(`${colors.yellow('Failed to fetch installed AVDs.')} Please try again.`);
```
Comment (garg3133) [inline src/commands/android/subcommands/uninstall/avd.ts:21]: We should also provide a command here that the user can run to list all the installed AVDs (probably using the dot command?).

Also, do you think it'd be worth adding a `list` subcommand to the tool, to list various things like `list devices` to list all the connected devices and `list avd` to list all the avds?

File: src/commands/android/subcommands/uninstall/avd.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    Logger.log();
+    Logger.log(`Deleting ${colors.cyan(avdName)}...\n`);
+
+    const deleteStatus = await execBinaryAsync(avdmanagerLocation, 'avdmanager', platform, `delete avd --name '${avdName}'`);
+
+    if (deleteStatus?.includes('deleted')) {
+      Logger.log(`${colors.green('AVD deleted successfully!')}`);
+
+      return true;
+    }
```
Comment (garg3133) [inline src/commands/android/subcommands/uninstall/avd.ts:43]: There should also be an else block here with `Something went wrong` and a way to verify if the AVD was deleted, similar to what you did in install AVD PR.

Comment (garg3133) [reply src/commands/android/subcommands/uninstall/avd.ts:43]: We could also output the `deleteStatus` in that case for the user to debug better.

File: src/commands/android/subcommands/uninstall/avd.ts (hunk 1 above)
Comment (itsspriyansh) [reply src/commands/android/subcommands/uninstall/avd.ts:21]: Sounds like a good idea. We can add a subcommand for listing devices.

**Comment by itsspriyansh** [comment]
@garg3133 PR updated. 
And PR raised for listing devices #66

**Pull request merged**